cmake/modules/OmrPlatform.cmake

```diff
@@ -56,13 +56,13 @@ macro(omr_platform_global_setup)
 
 	set(OMR_PLATFORM_GLOBALLY_INITIALIZED 1)
 
+	add_compile_options(${OMR_PLATFORM_COMPILE_OPTIONS})
```

Comment:
This will also add these flags to the assembler line.  On some platforms that will not work.

Reply:
@rwy0717 I believe this will break PPC LE.